nb2workflow/deploy.py

```diff
@@ -298,7 +300,37 @@ def _build_with_kaniko(git_origin,
         return container_metadata
 
 
-def _build_with_docker(git_origin, 
+def _extract_resource_requirements(local_repo_path):
+    from oda_api.ontology_helper import Ontology
+    from nb2workflow.nbadapter import NotebookAdapter
+    # TODO: replace with master version when PR is approved
+    ontology_URL = "https://raw.githubusercontent.com/oda-hub/ontology/storage_resource_annotations/ontology.ttl"
```

Comment:
Another option will be to make it configurable with environment variable (as long as no config file is used by this package).

Then we may either hardcode the default value to be `https://odahub.io/ontology/ontology.ttl` or disable this part if it's not defined (with the warning). Not sure what's better

Reply:
@dsavchenko what if we just add an extra parameter ontology_URL to deploy function with the default value "https://odahub.io/ontology/ontology.ttl" ? In my opinion it would be more transparent for user than using the environment variable.

Reply:
OK, I agree. As a function argument and also the cli argument.

Reply:
Done